src/style.css

```diff
@@ -1,3 +1,103 @@
 body {
   margin: 0;
+  --link-hover-color: #00acdc;
+  --link-active-color: var(--link-hover-color);
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: relative;
+  left: 50px;
+}
+
+.logo__img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  margin: 0;
+  position: relative;
+  right: 50px;
+  display: flex;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.nav__item {
+  /* padding: 0 22px 0 23px 0; */
+  /* border: 1px solid red; */
+}
+
+.nav__link {
+  display: flex;
```

Comment:
For what do you provide this CSS property in every selector ?

Reply:
Not fixed. Display flex is actually redundant here. Set the correct line-height to center it vertically